.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - make init
 script:
   - make build test lint coverage
+  - make docker-all
```

Comment:
Ah, nevermind. I misinterpreted GH diff (which is why the code looked OK to me at first) but then confused myself.

I am very much OK with the way things are. Sorry about the confusion.